cmd/bootnode/main.go

```diff
@@ -99,6 +100,7 @@ func main() {
 	versionFlag := flag.Bool("version", false, "Output version info")
 	verbosity := flag.Int("verbosity", 5, "Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail (default: 5)")
 	logConn := flag.Bool("log_conn", false, "log incoming/outgoing connections")
+	maxConnPerIP := flag.Int("max_conn_per_ip", 2, "max connections number for same ip")
```

Comment:
The default of 2 may be too low? We don't want to affect the normal operations of validators by this change. This change should only be targeted at absolute malicious behaviors.

Reply:
How about adjusting the default value to 10? It is sufficient for 10 nodes to use the same external network address